src/Mutation.tsx

```diff
@@ -184,19 +184,19 @@ class Mutation<TData = any, TVariables = OperationVariables> extends React.Compo
       });
   };
 
-  private mutate = (options: MutationOptions<TData, TVariables>) => {
+  private mutate = (options: MutationOptions<TData, TVariables> = {}) => {
     const {
       mutation,
       variables,
       optimisticResponse,
       update,
       context = {},
-      awaitRefetchQueries = false,
+      awaitRefetchQueries = (this.client.defaultOptions.mutate && this.client.defaultOptions.mutate.awaitRefetchQueries) || false,
```

Comment:
I don't think this is a great idea, since the default options are being applied inside apollo-client. Instead, you should just remove the default value for `awaitRefetchQueries` and only pass it into `client.mutate` if it is `!== undefined`. Then the default option will be applied inside the client.

Reply:
Hmm just assumed this was defaulted  for a reason so I kept it that way, will change then.

Reply:
It defaults to `false` inside the query manager anyway, so thats alright.

Reply:
The defaultOptions aren't being applied it seems... Test fails if I remove it

Reply:
Also gets defaulted here, so how can it respect the defaultOptions then? https://github.com/apollographql/apollo-client/blob/master/packages/apollo-client/src/core/QueryManager.ts#L127

Reply:
As I wrote, you also need to make sure not to pass it into `mutate` if its `undefined`, as that counts as the option being set.
```
const a = {foo: "bar"}
const b = {foo: undefined}
const c = {...a, ...b}
// c will be {foo: undefined}
```

Reply:
Yes but it is getting defaulted anyway as linked there, so when passing undefined it results in false anyway. When my defaultOptions is true it won't get respected since client makes no implication of respecting defaultOptions...

If it defaults an argument it has to make an assumption or it can't merge it with defaultOptions